RecoEgamma/EgammaTools/plugins/EGExtraInfoModifierFromDB.cc

```diff
@@ -364,118 +399,126 @@ namespace {
 }
 
 void EGExtraInfoModifierFromDB::modifyObject(reco::GsfElectron& ele) const {
+
   // regression calculation needs no additional valuemaps
 
   const reco::SuperClusterRef& the_sc = ele.superCluster();
   const edm::Ptr<reco::CaloCluster>& theseed = the_sc->seed();
   const int numberOfClusters =  the_sc->clusters().size();
   const bool missing_clusters = !the_sc->clusters()[numberOfClusters-1].isAvailable();
-
-
-  //skip HGCal for now
-  if( theseed->seed().det() == DetId::Forward ) return;
+  
   if( missing_clusters ) return ; // do not apply corrections in case of missing info (slimmed MiniAOD electrons)
+
+  const bool iseb = ele.isEB();  
+  edm::Handle<EcalRecHitCollection > ecalRecHits;
+  if (iseb) ecalRecHits = ecalRecHitsEB_ ;
+  else      ecalRecHits = ecalRecHitsEE_ ;
+  if ( !ecalRecHits.isValid() ) return;
   
-  std::array<float, 33> eval;  
+  Int_t N_SATURATEDXTALS  = 0;
```

Comment:
please use base C++ types.
Also, all caps for a variable is a bad style, it's usually used for compiler macros and some constants.